### DISCUSSION TITLE
[docs]:I think some examples have bugs in the documentation.

### DISCUSSION THREAD

**Discussion by RSS1102** [react-hook-form/react-hook-form#9406] [General] [answered]
I found that there may be many bugs in the document. I don't want to provide many `Codes and boxes/Expo snacks` (forgive my laziness).
Expo :
1. https://react-hook-form.com/api/useform/reset
I don't think there is any other need for `useState` to rely on listening, right?
![image](https://user-images.githubusercontent.com/81673017/203158126-a87f3738-fda2-4dcc-a800-83800ffac52f.png)

2. https://react-hook-form.com/api/usecontroller
Is a duplicate `name` declared here?
![image](https://user-images.githubusercontent.com/81673017/203159149-9c4fd2cd-c9b7-4002-885b-751623878e2d.png)

---
@bluebill1049 
I just started using the `react-book-form` today. If it's my mistake, please forgive me。

**Comment by bluebill1049** [comment, marked as answer]
thanks, please feel free to send PR next time as well.

**Reply by bluebill1049** [reply]
https://github.com/react-hook-form/documentation/commit/e805246684a1edcb428db700312c140245051b58

**Reply by RSS1102** [reply]
of couse ,It would be my pleasure to send PR, but I think that as a first-time user, I should first ask the author @bluebill1049 for advice   (rather than what I think is correct).:smile:
